### PR TITLE
Not for submission

### PR DESCRIPTION
**Pull request by amanda-tarafa**
Addind SchemaUpdateOptions, Flags Enum method in EnumMap and tests.

### CODE REVIEW

File: apis/Google.Cloud.BigQuery.V2/Google.Cloud.BigQuery.V2/SchemaUpdateOption.cs [hunk 1]
```diff
@@ -0,0 +1,37 @@
+﻿// Copyright 2017 Google Inc. All Rights Reserved.
```
Comment (chrisdunelm) [inline apis/Google.Cloud.BigQuery.V2/Google.Cloud.BigQuery.V2/SchemaUpdateOption.cs]: 2018? :)

File: apis/Google.Cloud.BigQuery.V2/Google.Cloud.BigQuery.V2/EnumMap.cs [hunk 2]
```diff
@@ -71,6 +74,35 @@ internal static string ToApiValue(T value, string paramName = "value")
             throw new ArgumentException($"Value {value} is undefined in {typeof(T).Name}", paramName);
         }
 
+        /// <summary>
+        /// Similar to <see cref="ToApiValue(T, string)"/> but for used with Flags enum.
+        /// </summary>
+        /// <param name="value">An enum value, possibly flagged.</param>
+        /// <returns>A set of string that contains the api value corresponding to each
+        /// of the flags set in <paramref name="value"/>.</returns>
+        internal static ISet<string> ToApiValues(T value)
```
[the remark this answers is not in the document]
Comment (amanda-tarafa) [reply apis/Google.Cloud.BigQuery.V2/Google.Cloud.BigQuery.V2/EnumMap.cs]: OK, let me try something like this then.

File: apis/Google.Cloud.BigQuery.V2/Google.Cloud.BigQuery.V2/SchemaUpdateOption.cs (hunk 1 above)
Comment (amanda-tarafa) [reply apis/Google.Cloud.BigQuery.V2/Google.Cloud.BigQuery.V2/SchemaUpdateOption.cs]: Yes :)

**Comment by amanda-tarafa** [comment]
Closing as this was not for submission.

**Pull request closed without merging**